swagger-spec/nodes/contributor_definition.yaml

```diff
@@ -74,4 +74,4 @@ example:
       user:
         data:
           type: users
-          id: '{user_id}'
+          id: 'guid0'
```

Comment:
the zeroth guid, I changed it from brackets, because there are valid examples of attributes all around it, so not putting a valid guid looks wrong in context. Look at other `examples:` guid0 is a common bit of syntax.